README.md

```diff
@@ -42,9 +42,13 @@ Before extending into new frontiers, we need to improve the loaders API enough t
 
 - [x] Provide a way to register loaders from application code, such as `import { register } from 'node:module'`. https://github.com/nodejs/node/pull/46826, https://github.com/nodejs/node/pull/48559.
 
-- [ ] Replace `globalPreload` hook with new `initialize` hook; update `register` to preserve the communications channel from that hook so that we continue to provide a way to communicate between loaders code and application code. See https://github.com/nodejs/loaders/discussions/124#discussioncomment-5735397 and https://github.com/nodejs/loaders/issues/147. https://github.com/nodejs/node/pull/48842.
+- [x] Replace `globalPreload` hook with new `initialize` hook; update `register` to preserve the communications channel from that hook so that we continue to provide a way to communicate between loaders code and application code. See https://github.com/nodejs/loaders/discussions/124#discussioncomment-5735397 and https://github.com/nodejs/loaders/issues/147. https://github.com/nodejs/node/pull/48842.
 
-- [ ] Support loading source when the return value of `load` has `format: 'commonjs'`. See https://github.com/nodejs/node/issues/34753#issuecomment-735921348 and https://github.com/nodejs/loaders-test/blob/835506a638c6002c1b2d42ab7137db3e7eda53fa/coffeescript-loader/loader.js#L45-L50. https://github.com/nodejs/node/pull/47999.
+- [ ] Remove `globalPreload` hook.
```

Comment:
Not sure about that. We might do it in the future, but it's not a condition for stability